docker-compose.yml

```diff
@@ -10,3 +10,52 @@ services:
       - "9966:9966"
     entrypoint: yarn
     command: start
+
+  django:
+    image: echolocator
+    environment:
+      - POSTGRES_HOST=database
+      - POSTGRES_PORT=5432
+      - POSTGRES_USER=echolocator
+      - POSTGRES_PASSWORD=echolocator
+      - POSTGRES_DB=echolocator
+      - DJANGO_ENV=Development
+      - DJANGO_SECRET_KEY=secret
+      - DJANGO_LOG_LEVEL=INFO
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    volumes:
+      - ./backend:/usr/local/src/backend
+      - $HOME/.aws:/root/.aws:ro
+    working_dir: /usr/local/src/backend
+    depends_on:
+      database:
+        condition: service_healthy
+    command:
+      - "-b :8081"
+      - "--reload"
+      - "--timeout=90"
+      - "--access-logfile=-"
+      - "--error-logfile=-"
+      - "--log-level=debug"
+      - "echo.wsgi"
+    ports:
+      - "8081:8081"
```

Comment:
I saw the requirement to "add to team ports document"- could you let me know how I can find the doc?

Reply:
Sounds good. I ended up using port `8085` for the Django server. Here is the PR for updating the docs: https://github.com/azavea/geospatial-apps/pull/261